src/osx/dataview_osx.cpp

```diff
@@ -699,10 +701,20 @@ void wxDataViewCtrl::FinishCustomItemEditing()
 
 void wxDataViewCtrl::AdjustAutosizedColumns() const
 {
-  if ( m_ModelNotifier )
+  if (!IsFrozen() && m_ModelNotifier )
```

Comment:
It looks either this check or the one in `AdjustAutosizedColumns()` is superfluous. Or do we really need both somehow?

Reply:
The one in `wxOSXDataViewModelNotifier::AdjustAutosizedColumns()` is needed - that method can be (and often is) called directly from the notifier.

This one does seem superfluous.